src/main/scala/com/mozilla/telemetry/streaming/EventsToAmplitude.scala

```diff
@@ -187,7 +187,10 @@ object EventsToAmplitude extends StreamingJobBase {
     // validate config json
     val factory = JsonSchemaFactory.byDefault
     val schema = factory.getJsonSchema(asJsonNode(getMetaSchema))
-    schema.validate(asJsonNode(json))
+    val report = schema.validate(asJsonNode(json))
```

Comment:
Did this stop throwing an exception on non-validation? I could have _sworn_ I made it explicitly do that...

Reply:
`validate` *does* throws its own exception but really only when the json isn't parseable (or anything else it deems a "processing error"). Otherwise it'll spit out a ProcessingReport, which is an iterable of messages, which in this case is essentially the list of validation errors, so that return value has to be examined.